### ISSUE TITLE
Error while exporting after recent changes to the Github repo.

### ISSUE THREAD

**Issue by sanjay-shah** [Layer7-Community/graphman-client#28]
```
./graphman.sh export --using all --output mybundle.json
[info] exporting from default gateway
[info] unexpected data {
  "errors": [
    {
      "message": "Validation error of type FieldUndefined: Field 'services' in type 'Query' is undefined @ 'services'",
      "locations": [
        {
          "line": 2,
          "column": 5
        }
      ],
      "extensions": {
        "classification": "ValidationError"
      }
    },
    {
      "message": "Validation error of type FieldUndefined: Field 'policies' in type 'Query' is undefined @ 'policies'",
      "locations": [
        {
          "line": 8,
          "column": 5
        }
      ],
      "extensions": {
        "classification": "ValidationError"
      }
    },
    {
      "message": "Validation error of type FieldUndefined: Field 'federatedIdps' in type 'Query' is undefined @ 'federatedIdps'",
      "locations": [
        {
          "line": 26,
          "column": 5
        }
      ],
      "extensions": {
        "classification": "ValidationError"
      }
    },
    {
      "message": "Validation error of type FieldUndefined: Field 'federatedGroups' in type 'Query' is undefined @ 'federatedGroups'",
      "locations": [
        {
          "line": 29,
          "column": 5
        }
      ],
      "extensions": {
        "classification": "ValidationError"
      }
    },
    {
      "message": "Validation error of type FieldUndefined: Field 'federatedUsers' in type 'Query' is undefined @ 'federatedUsers'",
      "locations": [
        {
          "line": 32,
          "column": 5
        }
      ],
      "extensions": {
        "classification": "ValidationError"
      }
    },
    {
      "message": "Validation error of type FieldUndefined: Field 'ldapIdps' in type 'Query' is undefined @ 'ldapIdps'",
      "locations": [
        {
          "line": 56,
          "column": 5
        }
      ],
      "extensions": {
        "classification": "ValidationError"
      }
    },
    {
      "message": "Validation error of type FieldUndefined: Field 'simpleLdapIdps' in type 'Query' is undefined @ 'simpleLdapIdps'",
      "locations": [
        {
          "line": 92,
          "column": 5
        }
      ],
      "extensions": {
        "classification": "ValidationError"
      }
    },
    {
      "message": "Validation error of type FieldUndefined: Field 'policyBackedIdps' in type 'Query' is undefined @ 'policyBackedIdps'",
      "locations": [
        {
          "line": 95,
          "column": 5
        }
      ],
      "extensions": {
        "classification": "ValidationError"
      }
    },
    {
      "message": "Validation error of type FieldUndefined: Field 'internalIdps' in type 'Query' is undefined @ 'internalIdps'",
      "locations": [
        {
          "line": 98,
          "column": 5
        }
      ],
      "extensions": {
        "classification": "ValidationError"
      }
    }
  ]
}
```

**Comment by sanjay-shah** [comment]
Can someone please help me with this issue?

**Comment by muemi03** [comment]
Hi. what versions are you using ?
Gateway version ?
graphman-client version ?
I guess, they just don't match. but a Broadcom colleague should look at it, to correct or confirm my belief.
I am just a user. same as you :)
My understanding : The main branch contains ongoing dev work which is dependent to the graphman assertion version on the gateway itself. 
So, best is to use : 
graphman-client v1.0.00 with gateways v10.1-CR03 up to V11.0-CR01
graphman-client v1.1.00 starting with V11.0-CR02
And don't use the main branch, unless you know what you're doing :)
Regards
...Michael

**Comment by graju256** [comment]
main branch is used for active development to work with the future gateway releases (for now, it is ready for 11.1.00 gateway release).

**Comment by muthugomu** [comment]
> Hi. what versions are you using ? Gateway version ? graphman-client version ? I guess, they just don't match. but a Broadcom colleague should look at it, to correct or confirm my belief. I am just a user. same as you :) My understanding : The main branch contains ongoing dev work which is dependent to the graphman assertion version on the gateway itself. So, best is to use : graphman-client v1.0.00 with gateways v10.1-CR03 up to V11.0-CR01 graphman-client v1.1.00 starting with V11.0-CR02 And don't use the main branch, unless you know what you're doing :) Regards ...Michael

I am using GW 11.0.00.14811 build 14811 - Container GW

**Comment by muthugomu** [comment]
I downloaded Graphman 1.1 Release and now it works like a charm in 11.0.00.14811 build 14811 Gateway version.

**Comment by sanjay-shah** [comment]
> main branch is used for active development to work with the future gateway releases (for now, it is ready for 11.1.00 gateway release).

Thanks @graju256, didn't realize the main branch was for active development. Error has been fixed by using v1.1.00 release. Thanks for your prompt help. The issue can be closed.

**Comment by graju256** [comment]
> > Hi. what versions are you using ? Gateway version ? graphman-client version ? I guess, they just don't match. but a Broadcom colleague should look at it, to correct or confirm my belief. I am just a user. same as you :) My understanding : The main branch contains ongoing dev work which is dependent to the graphman assertion version on the gateway itself. So, best is to use : graphman-client v1.0.00 with gateways v10.1-CR03 up to V11.0-CR01 graphman-client v1.1.00 starting with V11.0-CR02 And don't use the main branch, unless you know what you're doing :) Regards ...Michael
> 
> I am using GW 11.0.00.14811 build 14811 - Container GW and I am facing the same issue.
> 
> BTW, how do we know the graphman-client version? I do not see any command to list the graphman version? Appreciate if you could help me with the command to get the graphman-client version?

Just run the graphman with no options. You will see a lot of info about it along with it's version.
